### PR TITLE
Update 2.5

### PR DESCRIPTION
**Pull request by YvonneMa2001**
My version of exercise 2. Basically the same, different wording in explanation. Have an question about Q2, please see it when you guys are available.

### CODE REVIEW

File: 2.5 [hunk 1]
```diff
@@ -1,38 +1,42 @@
...
-# The code is littered with errors, including misspelling,
-# improper capitalization, and missing punctuation.
+ggplot(
+  data = mpg,
+  mapping = aes(x = displ, y = hwy)) +
+  geom_point()+
+  geom_smooth(method = "lm")
+
+# Do we have to do "  geom_smooth(aes(x = displ, y = hwy), method = "lm")"?
+# can we omit aes(x,y) inside geo_smooth as above?
```
Comment (epticrinduke) [inline 2.5:32]: Because mapping wasn't specified at the global layer, it needs to be specified for all sublayers.

**Review by epticrinduke — COMMENTED** [review]
I responded to your question in #2 via line comment. Would you like me to add the additional explanation you had in #1 to the main branch?

**Comment by YvonneMa2001** [comment]
> I responded to your question in #2 via line comment. Would you like me to add the additional explanation you had in #1 to the main branch?

**Comment by YvonneMa2001** [comment]
> I responded to your question in #2 via line comment. Would you like me to add the additional explanation you had in #1 to the main branch?

I see, thank you. I think it's just different wording, so the explanation we have now is good.

**Pull request closed without merging**